pandas/plotting/_matplotlib/timeseries.py

```diff
@@ -185,11 +185,10 @@ def _get_ax_freq(ax: Axes):
     return ax_freq
 
 
-def _get_period_alias(freq) -> str | None:
+def _get_period_alias(freq: BaseOffset | str) -> str | None:
```

Comment:
The pyi is typed as `timedelta | BaseOffset | str` and the docstring says `to_offset` takes timedeltas as well. Should that be included here?

Reply:
Thanks, yes `timedelta` should be allowed.